crates/avalanche-types/tests/rpc/common.rs

```diff
@@ -118,25 +113,26 @@ impl Handle for TestHandler {
         req: &Bytes,
         _headers: &[Element],
     ) -> std::io::Result<(Bytes, Vec<Element>)> {
-        let de_request: MethodCall = serde_json::from_slice(req).map_err(|e| {
+        let de_request: Request = serde_json::from_slice(req).map_err(|e| {
             io::Error::new(
                 io::ErrorKind::Other,
                 format!("failed to deserialize request: {e}"),
             )
         })?;
-
-        let json_str = serde_json::to_string(&de_request).map_err(|e| {
-            io::Error::new(
-                io::ErrorKind::Other,
-                format!("failed to serialize request: {e}"),
-            )
-        })?;
-
-        match self.handler.handle_request(&json_str).await {
-            Some(resp) => Ok((Bytes::from(resp), Vec::new())),
-            None => Err(io::Error::new(
+        
+        let bruh = (de_request.params,);
+
```

Comment:
Please change the name of the variable to something that's more appropriate

Reply:
Done